.eslintrc.js

```diff
@@ -107,6 +107,8 @@ module.exports = {
     {
       files: ['packages/*/src/**/*.js'],
       rules: {
+        // Array constructor allows us to leverage defensive coding
+        '@typescript-eslint/no-array-constructor': 'off',
```

Comment:
how?

Reply:
I take it back. It doesn't. fixed: 4d8e36de4